src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public void enterAllSongs() {
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
+        WebElement allSongs = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a[href='#!/songs']")));
+        allSongs.click();
+
+        // WebElement allSongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
+        //allSongs.click();
+
+        //Thread.sleep(1000);
+        // allSongs.click();
+        //
+    }
+
+    public void selectSong() {
+        WebElement song = driver.findElement(By.xpath("//tr[@class='song-item']"));
+        song.click();
+
+
+    }
+
+    public void enterButtonPlaySong() {
+        WebElement buttonPlaySong = driver.findElement(By.xpath("//span[@title='Play or resume']"));
+        Actions actions = new Actions(driver);
+        actions.click(buttonPlaySong).perform();
+    }
+
+
+    public boolean isDisplayedPlayingSong() {
+
+        WebElement songIsPlaying = driver.findElement(By.cssSelector("[data-testid = 'sound-bar-play']"));
+        return songIsPlaying.isDisplayed();
+
+
+    }
+
+    //Methods to rename a playlist
+    public void doubleClickOnPlaylist() {
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(1));
+
+        Actions actions = new Actions(driver);
+        WebElement myPlayList = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("/html/body/div/div/div/nav/section[2]/ul/li[3]/a")));
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.
For example, here is the locator (Copy > Copy Xpath): ` //*[@id="sidebar"]/section[1]/ul/li[3]/a`

BUT **I recommend not using plugins or copying the xpath** from the page code using a browser or web developer.
For example, here is the **correct locator:**  `//a[@class='songs']`

It is important to mention that the following principle must be held in deciding locators: the locator must be

**unique**
**readable** 
**maintainable**